identifiers/country-gb/countries.csv

```diff
@@ -0,0 +1,6 @@
+id,name
+ocd-division/country:gb,United Kingdom
+ocd-division/country:gb/country:gb-eng,England
+ocd-division/country:gb/country:gb-nir,Northern Ireland
+ocd-division/country:gb/country:gb-sct,Scotland
+ocd-division/country:gb/country:gb-wls,Wales
```

Comment:
Official there are no iso 3166-2 codes for these countries. they are all "gb".
For this case thay have a iso 3166-2:gb code
https://en.wikipedia.org/wiki/ISO_3166-2:GB
and there they have these codes above.

But for me it would be no problem to remove the prefix "gb-" for these countries, if you guys think so too?

Reply:
Done, "gb-" removed. And also part instead of country